src/main/java/Collectible.java

```diff
@@ -0,0 +1,16 @@
+public class Collectible extends Item {
+
+    /**
+     * Creates a new Collectible object.
+     * @param collectibleType the type of this Collectible.
+     */
+    public Collectible(String collectibleType){
+        super(collectibleType, 0);
+    }
+
+    public void setCollectibleNum(int collectibleNum){
+        int difference = collectibleNum - this.getNum();
```

Comment:
I don't think this is right
From CollectibleInventory, if I call setCollectibleNum(50) to increase the num by 50.
and getNum returns 20
difference = -30
So addnum would be the original number - 30

Try doing things like 
SetAmount(int x)
current amount += x

the function can both handle increasing or decreasing of collectible

Reply:
The method sets the collectible.num to the int passed into the method. 
Essentially collectible.num becomes collectibleNum.

So, if originally collectible.num = 10 and you wanted to increase it by 50, you could do collectible.addNum(50) or you could do collectible.setCollectibleNum(60). Both methods would return 60. 

However, I see what you're saying about having multiple methods to increase/decrease amounts, I'll combine the addNum() and decreaseNum() in Item.